src/bigscreenplayer.js

```diff
@@ -95,43 +100,26 @@ function BigscreenPlayer() {
     }
   }
 
-  function deviceTimeToDate(time) {
-    return getWindowStartTime() ? new Date(convertVideoTimeSecondsToEpochMs(time)) : new Date(time * 1000)
-  }
-
-  function convertVideoTimeSecondsToEpochMs(seconds) {
-    return getWindowStartTime() ? getWindowStartTime() + seconds * 1000 : null
-  }
+  function bigscreenPlayerDataLoaded({ media, enableSubtitles }) {
+    const initialPresentationTime =
+      initialPlaybackTime == null ? undefined : convertPlaybackTimeToPresentationTimeInSeconds(initialPlaybackTime)
```

Comment:
I think this does what you want, I.e. if `initialPlaybackTime` is `null` OR `undefined`, make it undefined, otherwise convert; but seeing double equals set off my Javascript Safety Alarms, maybe worth a comment?